static/styles/subscriptions.css

```diff
@@ -1108,12 +1108,14 @@ h4.user_group_setting_subsection_title {
         }
     }
 
-    #subscription_overlay,
     #groups_overlay {
         .display-type {
             display: none;
         }
+    }
```

Comment:
@m-e-l-u-h-a-n I'm not sure whether the `group_overlay` block here should be removed as well; can you test that and submit a quick PR if appropriate?